content/staff-wellbeing/staff-wellbeing-policy.md

```diff
@@ -0,0 +1,246 @@
+
+* make sure that the efforts and successes of staff are recognised and celebrated 
+
+* produce calendars of meetings, deadlines and events so that staff can plan ahead and manage their workload 
+
+* provide resources to promote staff wellbeing, such as training opportunities 
+
+* promote information about and access to external support services 
+
+* organise extra support during times of stress, such as Ofsted inspections 
+
+* welcome suggestions and give feedback 
+
+## Role of the principal 
+
+Creating a positive and supportive atmosphere throughout the school, the principal will: 
+
+* operate an open-door policy for all staff 
+
+* ensure that all polices that affect staff wellbeing are adhered to and reviewed 
+
+* monitor staff attendance data 
+
+* appraise the governing board of issues to do with staff wellbeing 
+
+* consult with representatives from unions regarding staff wellbeing 
+
+## Role of the local governing body  
+
+The local governing body is expected to: 
+
+* make sure the academy is fulfilling its duty of care as an employer, such as by giving staff a reasonable workload and creating a supportive work environment 
+
+* monitor and support the wellbeing of the principal 
+
+* make decisions and review policies with staff wellbeing in mind, particularly in regard to workload 
+
+* be reasonable about the format and quantity of information asked for from staff as part of monitoring work 
+
+* ensure that resources and support services are in place to promote staff wellbeing 
+
+* ensure that staff are clear about the purpose of any monitoring visits and what information will be required from them 
+
+## Managing specific wellbeing issues 
+
+The academy will support and discuss options with any member of staff who raises wellbeing issues, such as if they are experiencing significant stress at the academy or in their personal lives.  
+
+Where possible, support will be given by line managers or senior staff.  
+
+At all times, the confidentiality and dignity of staff will be maintained.  
+
+Support could be given through: 
+
+* reassessing their workload and deciding what tasks to prioritise 
+
+* temporarily relieving of some duties 
+
+* giving staff time off to deal with a personal crisis 
+
+* arranging external support, such as counselling or occupational health services 
+
+* the flexible working hours policy – this policy aims to cater for the needs of all staff and wherever possible the principal supports requests from staff for reduced or part time working where personal circumstances mean that it would improve their work-life balance 
+
+* completing a risk assessment and following through with any actions identified 
+
+* phased return or altered hours after absence 
+
+## Academy wellbeing committee and plan 
+
+The academy will facilitate a wellbeing committee with members from across the staff whose role will be to develop an action plan to promote a range of wellbeing interventions. It will: 
```

Comment:
```suggestion
The academy will facilitate a wellbeing committee with members from across the staff. Their role will be to develop an action plan to promote a range of wellbeing interventions. It will: 
```